runner/src/server/views/partials/summary-row.html

```diff
@@ -6,7 +6,7 @@
     <dd class="govuk-summary-list__value">
         {% if item.value %}
             {% if item.type == 'FileUploadField' %}
-                Uploaded
+                {{item.name}}
```

Comment:
If I'm correct it looks like you're displaying the file name?

The text should be "Check your uploaded file is visible. If not, upload it again."

Reply:
@donshellywork The item.name was right all along. Apologies for the confusion. Adam Davies has already done the changes in the bug ticket I raised so this PR can now be closed with the ticket you originally had moved to Done.

https://github.com/communitiesuk/digital-form-builder/pull/59